src/views/enroll-factors/BarcodeView.js

```diff
@@ -25,7 +25,7 @@ export default View.extend({
       </div>\
       <div class="scan-instructions-qrcode-wrapper">\
           <div class="qrcode-wrap">\
-              <img data-se="qrcode" alt="qr code" class="qrcode-image" src="{{qrcode}}">\
+              <img data-se="qrcode" alt="{{altQRCode}}" class="qrcode-image" src="{{qrcode}}">\
```

Comment:
Is the `i18n` templating function (as used below in the anchors) available here?

Reply:
Yeah it would be as well but I opted for this approach because it was a little cleaner. Otherwise, this line would exceed our maximum length and I would have had to break the <img> into multiple lines which wasn't ideal (just my personal preference)